### ISSUE TITLE
Starting up REST API v7 throws a NotWritablePropertyException on "taskqueue.dir"

### ISSUE THREAD

**Issue by tdonohue** [DSpace/DSpace#2939]
**Describe the bug**
When starting up the REST API backend in Tomcat, the following error is returned when the DSpaceKernel starts up:
```
14:33:04.291 [localhost-startStop-1] ERROR org.dspace.servicemanager.DSpaceServiceManager - Unable to set param (taskqueue.dir) on service bean (curate) to: /dspace/ctqueues
org.springframework.beans.NotWritablePropertyException: Invalid property 'taskqueue.dir' of bean class [org.dspace.curate.CurationScriptConfiguration]: Nested property in path 'taskqueue.dir' does not exist; 
nested exception is org.springframework.beans.NotReadablePropertyException: Invalid property 'taskqueue' of bean class [org.dspace.curate.CurationScriptConfiguration]: 
Bean property 'taskqueue' is not readable or has an invalid getter method: Does the return type of the getter match the parameter type of the setter?
```
This error does **NOT** seem to affect the REST API.  It still starts up and functions (seemingly without additional errors)

**To Reproduce**
1. Download latest code from `main` branch (as of 2020-Aug-26) and build it
2. Start up Tomcat, and watch the Tomcat logs for the error.

**Related work**
* *Might* be related to #2820, as `CurationScriptConfiguration` was added during the refactor of Curation Tasks into the scripts & processes framework.

**Comment by tdonohue** [comment]
@jonas-atmire : You might want to be aware of this bug, which looks (to me) like it was accidentally caused by #2820 (your past PR).  We might want to consider whether this can be fixed as part of the follow-up work in #2934

**Comment by tdonohue** [comment]
Closing. This is no longer reproducible. I *think* it may have been solved by #2968 (or some other recently merged PR)
